liboptv/include/orientation.h

```diff
@@ -10,7 +10,7 @@
 #include "imgcoord.h"
 #include "trafo.h"
 
-#define COORD_UNUSED INFINITY
+#define COORD_UNUSED -1e10
```

Comment:
@yosefm this line fails the check_track.c test. changing it back to -999 or INFINITY works